packages/ragel.rb

```diff
@@ -0,0 +1,18 @@
+require 'package'
+
+class Ragel < Package
+  description 'ragel desc stub'
+  homepage 'http://www.colm.net/open-source/ragel/'
+  version '6.10'
+  source_url 'http://www.colm.net/files/ragel/ragel-6.10.tar.gz'
```

Comment:
`https://www.colm.net/files/ragel/ragel-6.10.tar.gz` works so can we use that instead?

Reply:
Yes!